.github/workflows/breaking-change-check.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       # Checkout the branch to be merged into in PR
       - name: Checkout base branch
         if: ${{ github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'breaking-change') }}
-        run: git checkout ${{ github.event.pull_request.base.ref }}
+        run: git checkout ${{ github.event.pull_request.base.sha }}
```

Comment:
I guess people would still need to merge main before then finally check in, so this one is to reduce unwanted errors before the merging?

Reply:
Correct, similar to snapshot tests, the check needs to be against the developers code only (i.e. before and after just their code). I've seen the breaking-changes label be added to many PRs just to pass CI and I believe its because they hadn't merged from main and confused why it failed when they have no breaking changes.